src/openapi/streaming/streaming.js

```diff
@@ -463,13 +463,16 @@ Streaming.prototype.createSubscription = function(serviceGroup, url, subscriptio
 };
 
 /**
- * Makes a subscription start
+ * Makes a subscription start.
  *
  * @param {saxo.openapi.StreamingSubscription} subscription - The subscription to start.
+ * @param {Boolean} modify - The flag indicating if subscription is an modification. 
+ *                           If true, subscription will keep pending unsubscribe followed by subscribe with modify flag set to true. 
+ *                           Oherwise, unsubscribe in such scenario will be skipped.
  */
-Streaming.prototype.subscribe = function(subscription) {
+Streaming.prototype.subscribe = function(subscription, modify) {
```

Comment:
I would rather an explicit modify method that took the new arguments.
In future this could take an option of whether to a) wait for unsubscribe b) use "patch"